examples/3d/user_defined_clipping.rs

```diff
@@ -0,0 +1,64 @@
+//! The Simple 3D Scene with user-defined clipping enabled.
```

Comment:
A bit more explanation here about exactly what this is and why it might be useful in your games would go a long way :) The user shouldn't feel compelled to go Google an explanation if this is their first time seeing it.